packages/browser-sdk/src/lib/react/VideoView.tsx

```diff
@@ -7,28 +7,37 @@ interface VideoViewSelfProps {
     mirror?: boolean;
     style?: React.CSSProperties;
     onResize?: ({ width, height, stream }: { width: number; height: number; stream: MediaStream }) => void;
+    onSetAspectRatio?: ({ aspectRatio }: { aspectRatio: number }) => void;
 }
 
 type VideoViewProps = VideoViewSelfProps &
     React.DetailedHTMLProps<React.VideoHTMLAttributes<HTMLVideoElement>, HTMLVideoElement>;
 
-export default ({ muted, mirror = false, stream, onResize, ...rest }: VideoViewProps) => {
-    const videoEl = useRef<HTMLVideoElement>(null);
+export default ({ muted, mirror = false, stream, onResize, onSetAspectRatio, ...rest }: VideoViewProps) => {
+    const videoEl = React.useRef<HTMLVideoElement>(null);
 
-    useEffect(() => {
-        if (!videoEl.current || !onResize) {
+    React.useEffect(() => {
+        if (!videoEl.current) {
             return;
         }
 
         const resizeObserver = new ResizeObserver(
             debounce(
                 () => {
                     if (videoEl.current && stream?.id) {
-                        onResize({
-                            width: videoEl.current.clientWidth,
-                            height: videoEl.current.clientHeight,
-                            stream,
-                        });
+                        if (onResize) {
+                            onResize({
+                                width: videoEl.current.clientWidth,
+                                height: videoEl.current.clientHeight,
+                                stream,
+                            });
+                        }
+                        const h = videoEl.current.videoHeight;
+                        const w = videoEl.current.videoWidth;
+
+                        if (w && h && w + h > 20 && onSetAspectRatio) {
```

Comment:
When skimming the first time I was looking for a `if (onSetAspectRation)` to know the prop was optional. Could be better to reverse the order here?

Reply:
done 👍